packages/core/src/material/PBRMaterial.ts

```diff
@@ -222,6 +228,84 @@ export class PBRMaterial extends PBRBaseMaterial {
     }
   }
 
+  /**
+   * The sheen layer intensity, from 0.0 to 1.0.
+   * @defaultValue `0`
+   */
+  get sheen(): number {
+    return this.shaderData.getFloat(PBRMaterial._sheenProp);
+  }
+
+  set sheen(value: number) {
+    if (!!this.shaderData.getFloat(PBRMaterial._sheenProp) !== !!value) {
+      if (value === 0) {
+        this.shaderData.disableMacro("MATERIAL_ENABLE_SHEEN");
+      } else {
+        this.shaderData.enableMacro("MATERIAL_ENABLE_SHEEN");
```

Comment:
Whether to turn on sheen requires a comprehensive consideration of color and intensity